src/layer/poi.js

```diff
@@ -7,6 +7,7 @@ var iconDefs = {
   hospital: "hospital",
   medical: ["doctors", "clinic"],
   parking: "parking",
+  school: "school",
```

Comment:
```suggestion
  school: ["kindergarten", "school", "college", "university"],
```

Reply:
As I mentioned in the OP, I do think that down the line these features should render separately, probably in icon but definitely in zoom level. But at this stage, I can see the rationale in having the spirit of getting more categories rendering at all.